src/css/style.css

```diff
@@ -66,34 +66,17 @@ header {
 }
 
 /* LIST ITEM SECTION */
-img {
-  width: 300px;
-  height: 310px;
-  object-fit: fill;
-}
-
 .list-items {
   height: 75vh;
-  margin-top: 25px;
+  width: 75vw;
+  margin-top: 15px;
   display: grid;
-  grid-template-columns: auto auto auto;
+  grid-template-columns: 300px 300px 300px 300px;
   gap: 15px;
   padding: 20px;
   overflow-y: scroll;
 }
```

Comment:
![Screenshot (3)](https://user-images.githubusercontent.com/56417904/218273442-d097d771-a7a7-4fef-9a4a-d754d7cf5b2a.png)
![Screenshot (4)](https://user-images.githubusercontent.com/56417904/218273527-b78ef7a2-b806-4e6c-892b-f58c6f65d7bc.png)


Kindly change the width of the items. I think it is better not to give a fixed width for the items as it creates overflow for small width devices.